db/deploy/computed_columns/application_gis_data.sql

```diff
@@ -0,0 +1,21 @@
+-- Deploy ccbc:computed_columns/application_gis_data to pg
+
+begin;
+
+create or replace function ccbc_public.application_gis_data(application ccbc_public.application) 
+returns ccbc_public.gis_data_item as
+$$
+    select arr.item_object as json_data from ccbc_public.gis_data as gd,
+    jsonb_array_elements(gd.json_data) with ordinality arr(item_object, position)
```

Comment:
Would this work without position here? Like

```suggestion
    jsonb_array_elements(gd.json_data) with ordinality arr(item_object)
```